app/main.py

```diff
@@ -1,8 +1,41 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.spouse = None
+        self.wife = None
+        self.husband = None
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people_list: list) -> list:
```

Comment:
Please simplify this method:

There is no need for the `self.spouse` parameter.
Create persons using list comprehension and store them in a variable.
Iterate over people_list and check if a person has a 'wife' or 'husband'. If they do, retrieve the person object and the spouse object from `Person.people` by their names and assign them accordingly.